### ISSUE TITLE
New JInternalFrame does not have keyboard focus

### ISSUE THREAD

**Issue by Sciss** [mgarin/weblaf#90]
When using internal frames and I open a new `JInternalFrame`, that new frame is visibly shown to have the focus (has the blue outline), but the keyboard focus is still in the previous component (e.g. the internal frame which had focus before). Only after a component in the new internal frame requests focus, the keyboard focus is actually transferred.
